lib/stats/stats-registry.c

```diff
@@ -355,6 +355,31 @@ stats_get_cluster(const StatsClusterKey *sc_key)
   return sc;
 }
 
+gboolean
+stats_remove_cluster(const StatsClusterKey *sc_key)
+{
+  g_assert(stats_locked);
+  StatsCluster *sc;
+
+  sc = g_hash_table_lookup(stats_cluster_container.dynamic_clusters, sc_key);
+  if (sc)
+    {
+      if (stats_cluster_is_orphaned(sc))
+        return g_hash_table_remove(stats_cluster_container.dynamic_clusters, sc_key);
+      return FALSE;
+    }
+
+  sc = g_hash_table_lookup(stats_cluster_container.static_clusters, sc_key);
+  if (sc)
+    {
+      if (stats_cluster_is_orphaned(sc))
+        return g_hash_table_remove(stats_cluster_container.static_clusters, sc_key);
+      return FALSE;
+    }
+
+  return FALSE;
```

Comment:
`return TRUE`

Reply:
my bad

Reply:
For reference, this function mimics the return value of https://docs.gtk.org/glib/type_func.HashTable.remove.html#return-value